app/views/products/index.html.haml

```diff
@@ -1,10 +1,28 @@
+= render 'partials/navbar'
```

Comment:
Aha... you render it on this page. Why here and not on the `application.html.haml`?